src/components/DataGraph/util.js

```diff
@@ -35,13 +40,92 @@ var parseC3Data = function(data) {
                     }
                     yAxisCount++;
                 }
-            }              
+            }          
```

Comment:
This probably just slipped by, but if you're fixing white space, best to fix it *all* the way.